lib/sinon.js

```diff
@@ -15,7 +15,7 @@ var apiMethods = {
     assert: require("./sinon/assert"),
     match: require("@sinonjs/samsam").createMatcher,
     restoreObject: require("./sinon/restore-object"),
-    spyCall: require("./sinon/call"),
+    spyCall: require("./sinon/proxy-call"),
```

Comment:
We should probably remove this. It's very old and I don't know why it is exposed. Maybe along with the deprecated `sinon.sandbox.create` in the next major.

Reply:
Good idea